templates/cli/lib/utils.js.twig

```diff
@@ -67,7 +67,13 @@ function showConsoleLink(serviceName, action, open, ...ids) {
 
     if (open) {
         const start = (process.platform == 'darwin' ? 'open' : process.platform == 'win32' ? 'start' : 'xdg-open');
-        cp.exec(`${start} ${url}`);
+
+        cp.exec(`${start} ${url}`, (err, stdout, stderr) => {
+            if (err !== null) {
+                console.log('\n');
+                error('Opening in default browser. ' + err)
```

Comment:
what exactly are we doing here?why are we saying opening in default browser and appending the error?

Reply:
When the browser failed to open the link for any reason the `error` function will preint
Error: Opening in default browser. ERROR MESSAGE